### ISSUE TITLE
Add accessibility information (to Design System accessibility statement)

### ISSUE THREAD

**Issue by m-green** [alphagov/govuk-frontend-docs#71]
We need to add accessibility information about this documentation to the [Design System accessibility statement](https://design-system.service.gov.uk/accessibility/) by 23 September 2020, explaining how/whether this documentation meets WCAG 2.1.

## Done when

- [x] (Dependency) Tech Docs Template (TDT) has been audited by DAC, and accessibility statement added about any design issues with TDT generally
- [x] We've added lines to the Design System accessibility statement about known accessibility issues in Frontend docs, and plans to fix them
- [x] We've added the missing date.

**Comment by m-green** [comment]
https://github.com/alphagov/govuk-design-system/pull/1342
